packages/iTwinUI-react/src/core/Footer/FooterItem.tsx

```diff
@@ -0,0 +1,47 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils';
+import cx from 'classnames';
+import '@itwin/itwinui-css/css/footer.css';
+
+export type FooterItemProps = {
+  /**
+   * Url of the item.
+   */
+  url?: string;
+  /**
+   * Title of the item.
+   */
+  title?: string;
+  /**
+   * Content of the footer item. If provided, `title` and `url` props are ignored.
+   */
+  children?: React.ReactNode;
+} & Omit<React.ComponentPropsWithRef<'li'>, 'children'>;
+
+/**
+ * Footer item. Recommended to use with `Footer`.
+ */
+export const FooterItem = (props: FooterItemProps) => {
+  const { url, title, children, className, ...rest } = props;
+  useTheme();
+
+  return (
+    <li className={cx('iui-legal-footer-item', className)} {...rest}>
+      {children ? (
+        children
+      ) : url ? (
+        <a href={url} target='_blank' rel='noreferrer'>
```

Comment:
Do we really need all this extra url handling in this component? children should be enough

Reply:
At first it was a very basic component with just a class name so I added url and title handling because it seemed kind of duplicated work to do that ourselves.

Reply:
but now there is no way to customize this link. What if i don't want `target=_blank` or `rel='noreferrer'` in my footer?

Reply:
Then use `children` and pass everything yourself.

Reply:
then why even handle this? plus i don't think those attributes make sense as defaults.

if it's just to reduce code in our component, we can make an internal helper.